src/main/java/io/ebean/config/dbplatform/sqlserver/SqlServerPlatform.java

```diff
@@ -19,23 +24,22 @@
   public SqlServerPlatform() {
     super();
     this.platform = Platform.SQLSERVER;
-    // effectively disable persistBatchOnCascade mode for SQL Server
-    // due to lack of support for getGeneratedKeys in batch mode
-    this.persistBatchOnCascade = PersistBatch.NONE;
     this.idInExpandedForm = true;
     this.selectCountWithAlias = true;
     this.sqlLimiter = new SqlServerSqlLimiter();
     this.basicSqlLimiter = new SqlServerBasicSqlLimiter();
     this.platformDdl = new SqlServerDdl(this);
     this.historySupport = new SqlServerHistorySupport();
-    this.dbIdentity.setIdType(IdType.IDENTITY);
-    this.dbIdentity.setSupportsGetGeneratedKeys(true);
-    this.dbIdentity.setSupportsIdentity(true);
+    dbIdentity.setIdType(IdType.SEQUENCE);
```

Comment:
We can't just flip to sequence here without breaking people ... meaning we need a plan (like a ServerConfig option) to allow people to migrate.

Absolutely we want SqlServer folks to move to sequences so we do want this ... just need to be mindful of people already somewhat committed to IDENTITY.